src/components/notion-blocks/video.tsx

```diff
@@ -0,0 +1,32 @@
+import styles from '../../styles/notion-block.module.css'
+import React from 'react';
+import YouTube, {YouTubeProps } from 'react-youtube';
```

Comment:
```suggestion
import YouTube, { YouTubeProps } from 'react-youtube';
```
{} の内側にはスペースを1個お願いします。

Reply:
行末のセミコロンは不要です。